bson/bsoncodec/default_value_encoders_test.go

```diff
@@ -259,6 +271,22 @@ func TestDefaultValueEncoders(t *testing.T) {
 					bsonrwtest.WriteString,
 					errors.New("ev error"),
 				},
+				{
+					"empty map/success",
+					map[string]interface{}{},
+					&EncodeContext{Registry: NewRegistryBuilder().Build()},
+					&bsonrwtest.ValueReaderWriter{},
+					bsonrwtest.WriteDocumentEnd,
+					nil,
+				},
+				{
+					"with interface/success",
+					map[string]myInterface{"foo": myStruct{1}},
```

Comment:
Do we not test that the byte output is correct in these tests?

Reply:
No, these tests check correctness with errors and the last invoked function on the bsonrwtest.ValueReaderWriter